transferui/src/test/java/com/hyperwallet/android/ui/transfer/viewmodel/ListTransferDestinationViewModelTest.java

```diff
@@ -86,36 +88,39 @@ public Object answer(InvocationOnMock invocation) {
                 callback.onTransferMethodListLoaded(null);
                 return callback;
             }
-        }).when(transferMethodRepository).loadTransferMethods(ArgumentMatchers.any(
+        }).when(mTransferMethodRepository).loadTransferMethods(ArgumentMatchers.any(
                 TransferMethodRepository.LoadTransferMethodListCallback.class));
 
         ListTransferDestinationViewModel viewModel = factory.create(ListTransferDestinationViewModel.class);
+        viewModel.init();
 
         assertThat(viewModel.getTransferDestinationList(), is(notNullValue()));
         assertThat(viewModel.getTransferDestinationList().getValue(), Matchers.<TransferMethod>hasSize(0));
     }
 
     @Test
     public void testGetTransferDestinationSection_returnsLiveData() {
-        assertThat(mModelToTest.getSelectedTransferDestination(), is(notNullValue()));
+        assertThat(mListTransferDestinationViewModel.getSelectedTransferDestination(), is(notNullValue()));
     }
 
     @Test
     public void testGetTransferDestinationError_returnsLiveData() {
-        assertThat(mModelToTest.getTransferDestinationError(), is(notNullValue()));
+        assertThat(mListTransferDestinationViewModel.getTransferDestinationError(), is(notNullValue()));
     }
 
     @Test
     public void testInitialization_callsRepository() {
-        verify(transferMethodRepository).loadTransferMethods(
+        mListTransferDestinationViewModel.init();
+        verify(mTransferMethodRepository).loadTransferMethods(
                 any(TransferMethodRepository.LoadTransferMethodListCallback.class));
     }
 
     @Test
     public void testSelectTransferDestination_callsRepository() {
```

Comment:
I know it was named like that before, but since we're working on this class, could you please rename it to testSelectTransferDestination_returnsTransferMethod as there aren't invocations to any repository

Reply:
Done